LMS/User/Models/User.php

```diff
@@ -112,4 +112,9 @@ public function billings(): HasMany
     {
         return $this->hasMany(Billing::class);
     }
+
+    public function getUsernameAttribute($value): string
+    {
+        return '@'.$value;
```

Comment:
Espaçamento entre concatenações também.
```suggestion
        return '@' . $value;
```